test/unit/converter.test.ts

```diff
@@ -0,0 +1,102 @@
+import { ObjectId } from "mongodb";
+import { convertResultToResource, convertResultToStudyPlan } from "../../src/utils/converter";
+import { BASE_URL } from "@utils/constants";
+
+const URL = `${BASE_URL}/api/files`;
```

Comment:
NIT: Can be FILES_ENDPOINT_URL

Reply:
was thinking the same thing, will change

Reply:
U2U, but I agree